packages/patternfly-4/src/pages/index.js

```diff
@@ -1,20 +1,215 @@
+import pf4hardware from '../images/PF4hardware.svg';
+import principles from '../images/PF4_WIREFRAME.png';
+import redhatLogo from '../images/RHLogo.svg';
 
 const IndexPage = () => (
   <Layout>
     <SEO title="Home" keywords={['gatsby', 'application', 'react']} />
-    <PageSection variant={PageSectionVariants.darker} style={{height: 'calc(100vh - 76px)'}}>
-      <Bullseye>
-        <img src={orb} alt="PatternFly logo" />
-      </Bullseye>
+    <PageSection variant={PageSectionVariants.light} className="pf4-c-background-image pf-u-display-flex pf-u-justify-content-center pf-u-align-items-center">
+      <Grid>
+        <GridItem sm={12} md={6} mdOffset={3} className="pf-u-text-align-center">
+          <TextContent>
+            <img src={orb} alt="PatternFly logo" />
+            <Title size="4xl" className="pf-m-white">Build better experiences with repeatable, scalable design.</Title>
+            <Title size="xl" className="pf-m-white pf-u-mb-md pf-u-mb-3xl-on-md">PatternFly 4 is an open source design system built to drive consistency and unify teams.</Title>
+            <Link to="/get-started/about">
+              <button type="button" aria-label="Get started with PatternFly" className="pf4-c-button__cta-primary pf-u-mr-0 pf-u-mr-sm-on-sm pf-u-mb-md pf-u-mr-xl-on-md">Get Started</button>
+            </Link>
+            <Link to="#AboutPatternFly4">
+              <button type="button" className="pf4-c-button__cta-outline" aria-label="Why PatternFly 4">Why PatternFly 4</button>
+            </Link>
+          </TextContent>
+        </GridItem>
+      </Grid>
+    </PageSection>
+    <PageSection variant={PageSectionVariants.light}>
+      <Grid>
+        <GridItem sm={12} md={4} mdOffset={4} className="pf-u-text-align-center">
+          <Title size="4xl" id="AboutPatternFly4" className="pf-u-mb-md">About PatternFly 4</Title>
+          <Title size="lg">We provide tools like design documentation, components, and code examples to make it possible for anyone to design and build responsive, accessible user experiences.</Title>
+        </GridItem>
+      </Grid>
+    </PageSection>
+    <PageSection variant={PageSectionVariants.light} className="pf4-c-background-lines">
+      <Grid gutter="md" className="pf-u-pb-3xl">
+        <GridItem sm={12}>
+          <Grid>
+            <GridItem sm={12} md={4} mdOffset={2}>
+              <TextContent>
+                <div className="pf-u-pb-2xl">
+                  <Text component={TextVariants.h2}>Make better decisions with clear guidance</Text>
+                  <Text component={TextVariants.p}>Design guidelines supply the foundation of the system. Get acquainted with our approach to icons, color, content, and more.</Text>
+                  <Text component={TextVariants.a} href="/design-guidelines/styles/icons" aria-label="view design guidelines" className="pf-u-pl-0 pf4-c-link__text">View design guidelines <CaretRightIcon />
```

Comment:
You should be able to change this href to `/design-guidelines` as it will redirect

Reply:
Will it redirect to the Icons page?

Reply:
At the moment yes, but this might change depending on what the first item in the Design Guidelines is. That is how it should be in this case, no?

Reply:
Good point - will be updated in next PR push.